lib/drivers/dexcomDriver.js

```diff
@@ -285,12 +286,19 @@ module.exports = function (config) {
     var all = [];
     var ctr = 0;
 
-    var format = 'iisbs';
-    var recordformat = ['systemSeconds', 'displaySeconds', 'glucose', 'trendArrow', 'crc'];
+    var format;
+    var recordformat;
+    debug('EGV header revision:',header.revision);
```

Comment:
On second thought, this is too chatty as it will appear for every EGV record - will remove.